builder/src/main/java/cz/xtf/builder/builders/pod/ContainerBuilder.java

```diff
@@ -271,30 +277,6 @@ public void addPreStopHandler(Handler handler) {
         preStopHandler = handler;
     }
 
-    private static class VolumeMount {
```

Comment:
I would keep changes minimal, it is used only in this class, I would keep it nested and not add any unnecessary changes. 

Definitely, making the new class public makes it part of public API which we don't want. Unless I'm missing something.

Reply:
unfortunately we need this class to be public in order to configure its `subPath` property, e.g.:
```
volumeMounts.add(
				new VolumeMount("data-dir", "/opt/server/standalone/data/ejb-xa-recovery", false, "ejb-xa-recovery"));
```

Reply:
Can you please elaborate? I still don't see it. Isn't it
```
ContainerBuilder.addVolumeMount("data-dir", "/opt/server/standalone/data/ejb-xa-recovery", false, "ejb-xa-recovery")
```
?

Reply:
it's actually in another project

Reply:
I've sent you the link on the chat since it's an internal repo

Reply:
btw, in the snipped you pasted there a missing `new VolumeMount(...`

Reply:
ok, looking at other code, now I understand ... but in this case, I would even extend it by one more signature 

```
public ContainerBuilder addVolumeMount(VolumeMount volumeMount) {
   this.volumeMounts.add(volumeMount);
   return this;
}
```

otherwise it doesn't make sense to me to make it part of public API because there is really no way to use it outside XTF.

Reply:
done! tnks for the suggestion